service/search/es.go

```diff
@@ -73,6 +81,11 @@ type esMessageDocUpdate struct {
 	HasAudio       bool        `json:"hasAudio"`
 }
 
+type esCreateIndexBody struct {
+	Mappings m `json:"mappings"`
+	Settings m `json:"settings"`
+}
+
 type m map[string]interface{}
```

Comment:
ちょっと関係ないかもしれないけど、ここもアップデートということで
```suggestion
type m map[string]any
```